src/media/js/views/homepage.js

```diff
@@ -6,6 +6,8 @@ define('views/homepage',
     var gettext = l10n.gettext;
     var appsModel = models('apps');
 
+    var $appContextmenu = document.getElementById('contextmenu').children[0];
```

Comment:
This isn't a jquery element so no need to name it with a `$` prefix. Using raw JS here is fine but if a race condition causes the element to not yet be found this will generate a JS error.